tgui/packages/tgui/interfaces/KitchenMachine.js

```diff
@@ -0,0 +1,110 @@
+import { classes } from 'common/react';
+import { useBackend } from '../backend';
+import { Box, Button, Section, Table, Flex, Icon, Dimmer } from '../components';
+import { Window } from '../layouts';
+
+export const KitchenMachine = (props, context) => {
+  const { data } = useBackend(context);
+  const { ingredients = [],
+          name
+        } = data;
+
+  return (
+    <Window title= {name} resizable>
```

Comment:
```suggestion
    <Window title={name} resizable>
```
does the window not automatically set the frame name to the obj's name?